### DISCUSSION TITLE
Motion detection on Hero8 recording issue

### DISCUSSION THREAD

**Discussion by orufa** [gopro/labs#779] [Q&A]
Hi 
When I set up my Hero8 with motion detection and put my hand infront to activate it start recording for 4-5 seconds and stop and start again with the 15 seconds I programmed it with. Anybody else having the same issue?

**Comment by dnewman-gpsw** [comment]
Share your command string.

**Reply by orufa** [reply]
Sorry I don't understand.. what is a command string?

**Reply by dnewman-gpsw** [reply]
QR Code are just strings, so the command you ran.  Underneath the QR Code you will see the string
![image](https://github.com/gopro/labs/assets/10856084/1a65b699-5a05-4132-a95b-f0f5f17ea6b0)

So the command in this example is ```!SM3D4H5!R```

**Reply by orufa** [reply]
I've customiced it in the QRControl app and can't find any string under the qr code..

**Reply by orufa** [reply]
I found out what I did wrong!  In the protune meny start i've hooked on 'now'  cause I haven't set any time.. after I hooked 'now' of; the camera only record 1 time.

**Reply by dnewman-gpsw** [reply]
It shows in the QRControl too (if you don't name it.)

![image](https://github.com/gopro/labs/assets/10856084/f2fccbb1-2cab-4d97-bce4-ec24359c4bb5)

No one can really help unless you share the command you have sent. If you only have the QR code, point your phone at it to get the string.

**Reply by orufa** [reply]
Hi   I erased the name and then the string shows!  Thanks for the tip [dnewman-gpsw]!

**Comment by orufa** [comment]
I found out what I did wrong!  In the protune meny start i've hooked on 'now'  cause I haven't set any time.. after I hooked 'now' of; the camera only record 1 time.

**Reply by dnewman-gpsw** [reply]
Okay, don't do that.

If you enable repeat, it will repeat the motion capture.

**Reply by dnewman-gpsw** [reply]
This is also why it is important to read the command, so you can see any mistakes you are making. The command is added below the QR Code so you can learn how it works, from there create more complex commands.

**Reply by orufa** [reply]
I enabled repeat too and now it works fine. Hope to nail a sparrowhawk in my garden. It have a favorit branch which I will hook up the gopro with external powerbank. 
The string:    mV0r4p60fL20oSMwAx . 5sMcNb0d . . .

**Reply by dnewman-gpsw** [reply]
Ahh.. Don't set everything at once with a repeat command.  While you can do this, you will slow the repeat as you can setting the camera up with every capture.  The camera remembers, so have one QR Code for your settings, and another for your actions (like motion capture.)

**Reply by orufa** [reply]
Thanks for the tip!  Actualy I don't need to set framerate and resolution via the QR since it already is set up with 4K-60. Yust have to wait for now for the rainy days to stop.. been raining every day here in the westcoast of Norway.

**Comment by orufa** [comment]
I've solved my issue with the motion detection. Big thanks to David for his consultance in this matter and very good tips otherwize too!
